storage/test/test_cloudfront.py

```diff
@@ -73,6 +73,7 @@ def test_direct_download(
         engine = CloudFrontedS3Storage(
             context,
             "cloudfrontdomain",
+            {},
```

Comment:
is it possible to add a test which tests this config? i.e pass in a `namespace` to the `get_direct_downaload_url` function and expect a certain response